pocketfft_hdronly.h

```diff
@@ -2989,8 +2989,8 @@ class arr_info
     stride_t str;
 
   public:
-    arr_info(const shape_t &shape_, const stride_t &stride_)
```

Comment:
Do you know why `shape_` should be treated differently from `stride_` here?

Reply:
Good catch! Indeed clangtidy warned me about stride_ too, I simply wasn't paying full attention and missed it. I will revise stride_ in another commit in this pull request.